BudgetServiceTests/BudgetCalculationsTests.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  BudgetCalculationsTests.swift
+//  BudgetServiceTests
+//
+//  Created by antuan.khoanh on 15/07/2022.
+//
+
+import XCTest
+@testable import bonsai
+
+class BudgetCalculationsTests: XCTestCase {
+
+    func testBudgetMoneyCanSpendToday() throws {
+        // g
+        
+        let budgetCalculations = makeSUT()
```

Comment:
Instead of `makeSUT()` I suggest to override setup() method of `XCTestCase` and make `BudgetCalculations` - property of the class

Reply:
+fixed